http-clients/apache5-client/src/test/java/software/amazon/awssdk/http/apache5/Apache5HttpClientWireMockTest.java

```diff
@@ -115,44 +115,43 @@ public void routePlannerIsInvoked() throws Exception {
         mockProxyServer.verify(1, RequestPatternBuilder.allRequests());
     }
 
-    //TODO : Handle this as a part of supporting CredentialsProvider for Apache 5.x
-    // @Ignore("Need to fix CredentialsProvider for Apache 5.x")
-    // @Test
-    // public void credentialPlannerIsInvoked() throws Exception {
-    //     mockProxyServer.addStubMapping(any(urlPathEqualTo("/"))
-    //                                            .willReturn(aResponse()
-    //                                                            .withHeader("WWW-Authenticate", "Basic realm=\"proxy server\"")
-    //                                                            .withStatus(401))
-    //                                            .build());
-    //
-    //     mockProxyServer.addStubMapping(any(urlPathEqualTo("/"))
-    //                                            .withBasicAuth("foo", "bar")
-    //                                            .willReturn(aResponse()
-    //                                                            .proxiedFrom("http://localhost:" + mockServer.port()))
-    //                                            .build());
-    //
-    //     CredentialsProvider credentialsProvider = CredentialsProviderBuilder.create()
-    //                                                                         .add(new AuthScope(null, 0),
-    //                                                                              new UsernamePasswordCredentials("foo",
-    //                                                                                                              "bar".toCharArray()))
-    //                                                                         .build();
-    //
-    //
-    //     SdkHttpClient client = Apache5HttpClient.builder()
-    //                                             .credentialsProvider(credentialsProvider)
-    //                                             .httpRoutePlanner(
-    //                                                 (request, context) ->
-    //                                                     new HttpRoute(
-    //                                                         new HttpHost("https", "localhost", mockProxyServer.httpsPort())
-    //
-    //                                                     ))
-    //                                             .buildWithDefaults(AttributeMap.builder()
-    //                                                                            .put(TRUST_ALL_CERTIFICATES, Boolean.TRUE)
-    //                                                                            .build());
-    //     testForResponseCodeUsingHttps(client, HttpURLConnection.HTTP_OK);
-    //
-    //     mockProxyServer.verify(2, RequestPatternBuilder.allRequests());
-    // }
+    @Test
```

Comment:
Should we have more types of functional tests? end to end tests? 
Again, I know these tests didnt exist for Apache 4, but we don't know what we don't know.

Reply:
Our current migration focus is to ensure the Apache 5.x SDK client maintains the same behavior as Apache 4.x. For new modification like RepeatableInputStreamRequestEntity, we have added detailed unit tests.
End to end in test will be done with another task which is in plan.
 Is there any specific functionality you'd like us to focus on at this stage?